src/mozilla_sec_eia/models/sec_eia_record_linkage/transform_eia_input.py

```diff
@@ -0,0 +1,107 @@
+"""Create an EIA input utilities table that's ready for record linkage with the SEC 10K companies."""
+
+import numpy as np
+import pandas as pd
+from dagster import AssetOut, multi_asset
+
+from mozilla_sec_eia.library.record_linkage_utils import (
+    fill_street_address_nulls,
+    transform_company_name,
+)
+from mozilla_sec_eia.models.sec_eia_record_linkage.sec_eia_splink_config import STR_COLS
+
+EIA_COL_MAP = {
+    "utility_name_eia": "company_name",  # TODO: should be linking to owner or operator name?
+    "address_2": "street_address_2",
+}
+
+
+# TODO: make Dagster inputs instead of reading from AWS?
```

Comment:
I wasn't sure how to structure this EIA module with Dagster. Should the PUDL table inputs be assets? Should there be read in from AWS? Should the resultant EIA table be an asset?